crates/ui/src/slider.rs

```diff
@@ -282,6 +284,12 @@ impl Slider {
         self
     }
 
+    /// Set whether to display the tooltip on the thumb, default: true
+    pub fn display_tooltip(mut self, display_tooltip: bool) -> Self {
+        self.display_tooltip = display_tooltip;
+        self
+    }
```

Comment:
This is not a good API.

If we want to have this option, we need to consider to choice a better one. 

We are better to find any reference like this feature in other UI frameworks.

Reply:
What API would you prefer? I couldn't find any UI framework that displays the current slider value as a tooltip, so I also couldn't find a fitting api for disabling this.  I'd really like to have this option though, the current implementation isn't very good as it doesn't even properly round the values.